templates/genres.html

```diff
@@ -0,0 +1,162 @@
+                    <input type="checkbox" name="genres" value="gospel" id="gospel" onclick="checkboxChange(this)"><label for="gospel">Gospel</label>
+                    <input type="checkbox" name="genres" value="reggae" id="reggae" onclick="checkboxChange(this)"><label for="reggae">Reggae</label>
+                    <input type="checkbox" name="genres" value="groove" id="groove" onclick="checkboxChange(this)"><label for="groove">Groove</label>
+                    <input type="checkbox" name="genres" value="indie" id="indie" onclick="checkboxChange(this)"><label for="indie">Indie</label>
+                </div>
+            </div>
+            <div id="catDiv">
+                <input type="checkbox" name="categories" value="Dance/Electronic" id="dance" type="button" class="catCheckbox" onclick="displaySubOptions('dance')">
+                <label for="dance" style="font-size: 2.5rem; font-weight: 800">Dance / Electronic</label>
+                <div id="dance_genres" style="display:none">
+                    <input type="hidden" id="dance_hidden">
+                    <input type="checkbox" name="genres" value="dancehall" id="dancehall" onclick="checkboxChange(this)"><label for="dancehall">Dancehall</label>
+                    <input type="checkbox" name="genres" value="afrobeat" id="afrobeat" onclick="checkboxChange(this)"><label for="afrobeat">Afrobeat</label>
+                    <input type="checkbox" name="genres" value="edm" id="edm" onclick="checkboxChange(this)"><label for="edm">EDM</label>
+                    <input type="checkbox" name="genres" value="party" id="party" onclick="checkboxChange(this)"><label for="party">Party</label>
+                    <input type="checkbox" name="genres" value="house" id="house" onclick="checkboxChange(this)"><label for="house">House</label>
+                    <input type="checkbox" name="genres" value="deep-house" id="deep-house" onclick="checkboxChange(this)"><label for="deep-houseDeep">Deep House</label>
+                    <input type="checkbox" name="genres" value="club" id="club" onclick="checkboxChange(this)"><label for="club">Club</label>
+                    <input type="checkbox" name="genres" value="disco" id="disco" onclick="checkboxChange(this)"><label for="disco">Disco</label>
+                    <input type="checkbox" name="genres" value="dubstep" id="dubstep" onclick="checkboxChange(this)"><label for="dubstep">Dubstep</label>
+                    <input type="checkbox" name="genres" value="drum-and-bass" id="drum-and-bass" onclick="checkboxChange(this)"><label for="drum-and-bass">Drum and Bass</label>
+                    <input type="checkbox" name="genres" value="tango" id="tango" onclick="checkboxChange(this)"><label for="tango">Tango</label>
+                </div>
+            </div>
+            <div style="width:100%; float:left">
+                <input type="submit" class="button" value="Continue">
+            </div>      
+        </form>
+        <script>
```

Comment:
This javascript implementation is great. I'd love if you could comment explanations for some of the functions to ensure understandability.

Reply:
Definitely, in my next pull request I'll add comments to my code for better readability and understanding.